frontend_server/README.md

```diff
@@ -0,0 +1,37 @@
+Frontend Server
+==============
+
+Frontend server is simple wrapper around Dart Frontend. It is a Dart application
+that compiles Dart source into Dart Kernel binary(.dill-file).
+
+Frontend server runs in two modes:
+ - immediate mode, where Dart source file name is provided as command line
+ argument;
+ - interactive mode, where communication is happening over stdin/stdout.
+
+ Interactive mode instructions
+ ---
+- ```compile <input.dart>```
+ Compiles <input.dart> Dart source file with Dart Frontend, replies with ```result``` response.
+- ```recompile <boundary-key>```
+```<path/to/updated/file1.dart>```
+```<path/to/updated/file2.dart>```
+```...```
+```<boundary-key>```
```

Comment:
seems like this should all be one block instead of 5 blocks (i.e. just start with triple-backtick then end with triple-backtick, rather than having each line in backticks)